gensim/utils.py

```diff
@@ -836,7 +837,7 @@ def run(self):
 
 
 if os.name == 'nt':
-    logger.info("detected Windows; aliasing chunkize to chunkize_serial")
```

Comment:
Please keep this line as info. Info is used for confirmation that things are working as expected.